src/server/scripts/OutdoorPvP/OutdoorPvPGH.cpp

```diff
@@ -25,12 +23,8 @@ bool OutdoorPvPGH::SetupOutdoorPvP()
     RegisterZone(GH_ZONE);
     SetMapFromZone(GH_ZONE);
 
-    if ((m_obj = new OPvPCapturePointGH(this)))
-    {
-        AddCapturePoint(m_obj);
-        return true;
-    }
-    return false;
+    AddCapturePoint(new OPvPCapturePointGH(this));
+    return true;
```

Comment:
you are changing the logic here

Reply:
@Helias I'm not, this was a mistake as the condition:

```cpp
if ((m_obj = new OPvPCapturePointGH(this)))
```
returns always true, because `new OPvPCapturePointGH(this)` is always true